### DISCUSSION TITLE
如果我可以自己做一个启动器，那会不会侵权呀

### DISCUSSION THREAD

**Discussion by RainbowTaro2022** [Meloong-Git/PCL#2810] [提问] [answered]
我大概考虑了一下，想做一个启动器了
但是又怕侵权，请大佬们想想办法出个主意

**Comment by burningtnt** [comment]
如果你直接抄 PCL2 的代码，那肯定是违反开原协议的

**Comment by MoYuan-CN** [comment, marked as answer]
根据仓库中的 [LICENSE](https://github.com/Hex-Dragon/PCL2/blob/main/LICENCE) 规定
**不允许**以本存储库的内容作为内容主体创建、发布、传播其衍生作品，且您**不可以**发布本存储库及其相关作品的修改版。
在满足以上条款，且**在可见位置标注内容来源（即本存储库链接）的前提下**，您可以使用或引用本存储库的小部分内容或是其实现思路。

---

特别注意，[此 LICENSE](https://github.com/Hex-Dragon/PCL2/blob/main/LICENCE)**不是**法律文件或开源协议，龙腾猫跃对本存储库的内容**保留所有权利**

**Reply by iPodtouch6557** [reply]
是否包括借鉴交互逻辑和布置（不是外观上的）

**Reply by MoYuan-CN** [reply]
好问题 ~~(LICENSE貌似没写)~~ 问问龙猫？

**Reply by iPodtouch6557** [reply]
行吧~
 @LTCatt

**Comment by RainbowTaro2022** [comment]
如果我不使用pcl2的代码呢？
我直接从国外启动器上抄呢？

**Reply by MoYuan-CN** [reply]
国外启动器？你可以访问他们的官网或 Github 仓库，寻找 LICENSE 等协议，若无，你可以询问作者，确保合法的使用资源

**Reply by RainbowTaro2022** [reply]
有一些启动器没有仓库的

**Reply by MoYuan-CN** [reply]
“你可以询问作者，确保合法的使用资源”

**Comment by WForst-Breeze** [comment]
在借鉴或照搬启动器作者源代码时需要留意开发者的相关要求或开源协议，如开发者明确了不允许，那么就是不可以，如果开发者没有公开明确说明，则需要直接咨询联系开发者，如果无法取得联系，那最好也不要这样做。在遵循原开发者意愿和要求的情况下制作启动器不构成侵权
